dot/rpc/subscription/websocket.go

```diff
@@ -278,6 +278,42 @@ func (c *WSConn) initBlockFinalizedListener(reqID float64, _ interface{}) (Liste
 	return bfl, nil
 }
 
+func (c *WSConn) initAllBlocksListerner(reqID float64, _ interface{}) (Listener, error) {
+	listener := new(AllBlocksListener)
+	listener.cancel = make(chan struct{})
+	listener.done = make(chan struct{})
+	listener.cancelTimeout = defaultCancelTimeout
+	listener.wsconn = c
+	listener.finalizedChan = make(chan *types.FinalisationInfo, DEFAULT_BUFFER_SIZE)
+	listener.importedChan = make(chan *types.Block, DEFAULT_BUFFER_SIZE)
```

Comment:
move to constructor `newAllBlocksListener`

Reply:
Done!